README.adoc

```diff
@@ -134,6 +134,15 @@ Type: `bool`
 
 Default: `false`
 
+==== [[input_verbose_helm_templates]] <<input_verbose_helm_templates,verbose_helm_templates>>
+
+Description: A boolean to enable/disable outputting Helm templates on the Terraform plan.  
```

Comment:
I'm not sure this feature is the main reason why secrets are shown in the plan. I'd rather say: "make sure no secrets appear in the Kubernetes manifests before setting this flag."

Reply:
Same applies to other READMEs.